pyartcd/pyartcd/pipelines/build_microshift_bootc.py

```
@@ -197,27 +212,121 @@ async def _run_pipeline(self):
+
+        # Discover which bootc variants exist in this group's build-data
+        variants = await self._get_bootc_variants()
+        self._logger.info("Discovered bootc variants for group %s: %s", self.group, variants)
+
+        # Build all variants in parallel
+        build_results = await asyncio.gather(
+            *(self._rebase_and_build_bootc(variant) for variant in variants),
+            return_exceptions=True,
+        )
+        builds: dict[str, KonfluxBuildRecord] = {}
+        for variant, result in zip(variants, build_results):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)